native/src/modules/fetcher/FetcherModule.js

```diff
@@ -1,45 +1,55 @@
 // @flow
 
+import type { Channel } from 'redux-saga'
 import NativeFetcherModule, { NativeFetcherModuleEmitter } from './NativeFetcherModule'
+import { eventChannel, END } from 'redux-saga'
 import { isEmpty } from 'lodash'
 
 export type TargetFilePathsType = { [path: string]: string }
 
 export type FetchResultType = { [path: string]: {| lastUpdate: string, url: string, errorMessage: ?string |} }
 
-export type ProgressCallbackType = (progress: number) => void
-
 class FetcherModule {
   // TODO NATIVE-264: Correctly handle already fetching
```

Comment:
Also: [IGAPP-217](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=14&view=detail&selectedIssue=IGAPP-217): Fix the error "Already fetching!" when switching cities

Reply:
That belongs to the existing comment. just had a look,, NATIVE-264 does not exist any more, It is renamed to  IGAPP-217
If corrected the Task number

Reply:
Ahhh right, thanks :+1: